### PR TITLE
adding known issue pod logs

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/16470

### CODE REVIEW

**Review by zlayne — APPROVED** [review]
lgtm

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -386,6 +386,22 @@ The link:https://argo-cd.readthedocs.io/en/stable/[Argo CD] integration with {pr
...
+=== Non-{ocp} managed clusters must have LoadBalancer enabled
+
+Both {ocp} and non-{ocp-short} clusters support the pod log feature, however non-{ocp-short} require `LoadBalancer` to be enabled to use the feature. Complete the following steps to enable `LoadBalancer`:
+
+. Cloud providers have different `LoadBalancer` configurations. Visit your cloud provider documentation for more information. 
+. Verify if `LoadBalancer` is enabled on your {product-title-short} by checking the `loggingEndpoint` in the status of `managedClusterInfo`. 
+. Run the following command to check if the `loggingEndpoint.IP` or `loggingEndpoint.Host` has a valid IP address or host name:
++
+----
+need command if that is how the user can check
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: @zlayne Is there a command that the user can run to view these parameters? Do you they access the parameters from the console?

Comment (zlayne) [reply release_notes/known_issues.adoc]: This should return the info the user will need:
```
oc get managedclusterinfo <clusterName> -n <clusterNamespace> -o json | jq -r '.status.loggingEndpoint'
```

@zhiweiyin318 Can you verify the doc info in this PR as well?

Comment (dockerymick) [reply release_notes/known_issues.adoc]: @zlayne Thank for this info !

**Comment by zhiweiyin318** [comment]
thanks @dockerymick . this looks great to me.

**Pull request merged**